src/utils/errors.ts

```diff
@@ -9,7 +9,7 @@ export class AccessTokenError extends Error {
   public code: string;
 
   constructor(code: string, message: string) {
-    super(message);
+    super(message) /* istanbul ignore next */;
```

Comment:
With ES5, code coverage gets reported incorrectly when using the `super` call inside a constructor. See https://github.com/microsoft/TypeScript/issues/13455

Reply:
Also see https://github.com/gotwarlost/istanbul/issues/690